### ISSUE TITLE
Pie chart skeleton is missing aria-label

### ISSUE THREAD

**Issue by r1shabhsharma** [carbon-design-system/carbon-charts#1984]
### Relevant package(s)

- [x] @carbon/charts
- [ ] @carbon/charts-svelte
- [ ] @carbon/charts-react
- [ ] @carbon/charts-angular
- [ ] @carbon/charts-vue
- [ ] Documentation website

### Carbon Charts Version

1.23.6

### Which bundler are you using?

None (UMD)

### What happened and what did you expect to happen?

Pie chart is missing aria-label field causing accessibility issues

### Chart data and options (automatically formatted so no need for backticks)

```js

```

### JavaScript console or build output (if relevant)

```shell

```

### IBM Application/Team (if relevant)

SCIS - UIHub

### What priority level would this be in your opinion?

P2 (Medium)
